### DISCUSSION TITLE
Does Quartz.net supports persistence with filebased job storage?

### DISCUSSION THREAD

**Discussion by Sairam-Sunkari** [quartznet/quartznet#2541] [Q&A] [answered]
I have a need to support, scheduling using Quartz.net without sqlserver or any database. So planning to use windows file as job storage.
Does Quartz.net has any file based job storage(provider )support or any body tried using any custom or open source file based storage along with Quartz?

Thanks in advance

**Comment by lahma** [comment, marked as answer]
SQLite support comes in the box.
